### PR TITLE
2.1 WG Approved Version

### PR DESCRIPTION
**Pull request by TelegramSam**
This creates the 2.1 version of the spec, and associated link changes.

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: docs/spec-files/title.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ DIDComm Messaging v2.x Editor's Draft
 **Specification Status:** DIF Ratified Specification
 
 **Latest Stable version:**
-  [identity.foundation/didcomm-messaging/spec/v2.0](https://identity.foundation/didcomm-messaging/spec/v2.0)
+  [identity.foundation/didcomm-messaging/spec/v2.0](https://identity.foundation/didcomm-messaging/spec/v2.1)
```
Comment (FabioPinheiro) [inline docs/spec-files/title.md:7]:
```suggestion
  [identity.foundation/didcomm-messaging/spec/v2.1](https://identity.foundation/didcomm-messaging/spec/v2.1)
```

File: docs/spec-files-v2.1-snapshot/title.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+DIDComm Messaging v2.1 Editor's Draft
+==================
+
+**Specification Status:** Working Group Approved
+
+**Latest Stable version:**
+  [identity.foundation/didcomm-messaging/spec/v2.0](https://identity.foundation/didcomm-messaging/spec/v2.1)
```
Comment (FabioPinheiro) [inline docs/spec-files-v2.1-snapshot/title.md:7]:
```suggestion
  [identity.foundation/didcomm-messaging/spec/v2.1](https://identity.foundation/didcomm-messaging/spec/v2.1)
```

**Review by FabioPinheiro — COMMENTED** [review]
LGTM

File: docs/spec-files-v2.1-snapshot/title.md (hunk 2 above)
Comment (FabioPinheiro) [reply docs/spec-files-v2.1-snapshot/title.md:7]: What about adding a link to the previous version?

**Comment by mccown** [comment]
All these updates look good and I think it's ready to go.

**Pull request merged**